homeassistant/components/recorder/statistics.py

```diff
@@ -239,6 +239,18 @@ def _configured_unit(unit: str, units: UnitSystem) -> str:
     return unit
 
 
+def clear_statistics(hass: HomeAssistant, statistic_ids: list[str]) -> None:
+    """Clear statistics for a list of statistic_ids."""
+    with session_scope(hass=hass) as session:
+        metadata_ids = _get_metadata_ids(hass, session, statistic_ids)
+        session.query(Statistics).filter(
```

Comment:
This is not necessary because we have this in recorder models:

```python
    metadata_id = Column(
        Integer,
        ForeignKey(f"{TABLE_STATISTICS_META}.id", ondelete="CASCADE"),
        index=True,
    )
```